docs/design/multi-surface-targeting.md

```
@@ -213,53 +225,60 @@ mechanism (overlap startup instead of paying it serially).
 // later: act in that tab by name
 "type": { "keys": ["…"], "surface": { "browser": "chrome", "window": "admin", "tab": "overview" } }
 ```
 
-## Closing surfaces — `closeSurface` (replaces `stopProcess`)
+## Closing surfaces: `closeSurface`, which replaces `stopProcess`
 
-The symmetric partner of `startSurface`. Takes a `surface` **reference**,
-progressive, never-fail-on-missing (inherits #385). Array form closes several.
+This is the symmetric partner of `startSurface`. It takes a `surface`
+**reference**, is progressive, and never fails on a missing surface, inheriting
+#385. The array form closes several.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)